feature_engine/estimator_checks.py

```diff
@@ -231,6 +238,17 @@ def check_get_feature_names_out(estimator):
                 "date2",
             ]
 
```

Comment:
We need to add same test as above (when input_features is None), but using pipe right here.

Reply:
Do you mean to add the following code in the `if` statement associated w/ `# selection transformers`?

```
# transformer is used in a pipeline
assert pipe.get_feature_names_out(
    input_features=features
) == features
```